changelogs/changelog.yml

```diff
@@ -1,5 +1,62 @@
 ancestor: null
 releases:
+  2.3:
+    changes:
+      major_changes:
+      - Add support Pardus 21.x autoinstall
```

Comment:
Pardus also supports test according to above table.

Reply:
Pardus doesn't support all test cases. Marked with (*) in README.md